gen/calc.py

```diff
@@ -412,6 +441,8 @@ def validate_bootstrap_tmp_dir(bootstrap_tmp_dir):
               }                                         \
             ]}',
         'dcos_remove_dockercfg_enable': "false",
+        'minuteman_min_named_ip': '11.0.0.0',
+        'minuteman_max_named_ip': '11.0.0.254',
```

Comment:
11.255.255.255 is a broadcast address for the subnet but I got your point that the range should end with 11.255.255.254.